modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -605,6 +612,26 @@ private InternetAddress[] _toInternetAddresses(String string)
 		return internetAddresses.toArray(new InternetAddress[0]);
 	}
 
+	private void _validateEmailAddress(String emailAddress)
+		throws PortalException {
```

Comment:
@lucasmiranda0 please rename it to `_validateEmailAddresses(String emailAddressesString)`